sdk/data_kusto/src/operations/query.rs

```diff
@@ -0,0 +1,191 @@
+use crate::client::KustoClient;
+use azure_core::prelude::*;
+use azure_core::setters;
+use azure_core::{collect_pinned_stream, Response as HttpResponse};
+use futures::future::BoxFuture;
+
+type ExecuteQuery = BoxFuture<'static, crate::error::Result<KustoResponseDataSetV2>>;
+
+#[derive(Debug, Serialize, Deserialize)]
+struct QueryBody {
+    db: String,
+    csl: String,
```

Comment:
If we are going to use short names like this we should document what they need.

Reply:
added some docs.